packages/contractkit/src/wrappers/Election.ts

```diff
@@ -0,0 +1,220 @@
+import { eqAddress } from '@celo/utils/lib/address'
+import { zip } from '@celo/utils/lib/collections'
+import BigNumber from 'bignumber.js'
+import { Address, NULL_ADDRESS } from '../base'
+import { Election } from '../generated/types/Election'
+import {
+  BaseWrapper,
+  CeloTransactionObject,
+  proxyCall,
+  proxySend,
+  toBigNumber,
+  toNumber,
+  wrapSend,
+} from './BaseWrapper'
+
+export interface Validator {
+  address: Address
+  id: string
+  name: string
+  url: string
+  publicKey: string
+  affiliation: string | null
+}
+
+export interface ValidatorGroup {
+  address: Address
+  id: string
+  name: string
+  url: string
+  members: Address[]
+}
+
+export interface ValidatorGroupVote {
+  address: Address
+  votes: BigNumber
+  eligible: boolean
+}
+
+export interface ElectionConfig {
+  minElectableValidators: BigNumber
+  maxElectableValidators: BigNumber
+  electabilityThreshold: BigNumber
+  maxNumGroupsVotedFor: BigNumber
+}
+
+/**
+ * Contract for voting for validators and managing validator groups.
+ */
+export class ElectionWrapper extends BaseWrapper<Election> {
+  activate = proxySend(this.kit, this.contract.methods.activate)
+  /**
+   * Returns the minimum number of validators that can be elected.
+   * @returns The minimum number of validators that can be elected.
+   */
+  minElectableValidators = proxyCall(
+    this.contract.methods.minElectableValidators,
+    undefined,
+    toBigNumber
+  )
+  /**
+   * Returns the maximum number of validators that can be elected.
+   * @returns The maximum number of validators that can be elected.
+   */
+  maxElectableValidators = proxyCall(
+    this.contract.methods.maxElectableValidators,
+    undefined,
+    toBigNumber
+  )
+  /**
+   * Returns the current election threshold.
+   * @returns Election threshold.
+   */
+  electabilityThreshold = proxyCall(
+    this.contract.methods.getElectabilityThreshold,
+    undefined,
+    toBigNumber
+  )
+  validatorAddressFromCurrentSet = proxyCall(this.contract.methods.validatorAddressFromCurrentSet)
```

Comment:
Should this have a natspec?

Reply:
It does. All should have; i've merged a PR where i added docs for all wrapper methods; but it's hard to avoid other PRS to merge uncommented methods :P

While you are on it, make sure you change input param to `number` instead of `number|string`.

Basically you need to do:
```ts
validatorAddressFromCurrentSet: (index:number)=> Promise<Address> = proxyCall(this.contract.methods.validatorAddressFromCurrentSet, tupleParser<number,number>(identity))
```